web/src/api.ts

```diff
@@ -29,6 +29,31 @@ export const getCustomer = async (customerId: number): Promise<Customer> => {
   return res.json();
 };
 
+/**
+ * Logs in a customer using their
+ * @param customerData
```

Comment:
Add more explanation to the parameter?

Reply:
@chshda Omg 😅 Not sure how I missed it haha